services/db/01_user.sql

```diff
@@ -1,18 +1,3 @@
-/*!40000 ALTER TABLE `user` ENABLE KEYS */;
-UNLOCK TABLES;
\ No newline at end of file
+version https://git-lfs.github.com/spec/v1
```

Comment:
please undo this change

Reply:
Hi Professor, I am unable to modify this.

I downloaded the original file from your repo and added that to my local. However, I am not able to commit it as it says my branch is up to date with all the files. I tried to open this file locally and check what is inside it, it is the same as in your file. Can you please help me here?